specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2021-04-15/rbac.json

```diff
@@ -0,0 +1,573 @@
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "description": "Retrieves the list of all Azure Cosmos DB SQL Role Definitions.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "cosmos-db.json#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The list operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/SqlRoleDefinitionListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/sqlRoleAssignments/{roleAssignmentId}": {
+      "get": {
+        "operationId": "SqlResources_GetSqlRoleAssignment",
+        "x-ms-examples": {
+          "CosmosDBSqlRoleAssignmentGet": {
+            "$ref": "./examples/CosmosDBSqlRoleAssignmentGet.json"
+          }
+        },
+        "description": "Retrieves the properties of an existing Azure Cosmos DB SQL Role Assignment with the given Id.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/roleAssignmentIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "cosmos-db.json#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Role Assignment was successfully retrieved.",
+            "schema": {
+              "$ref": "#/definitions/SqlRoleAssignmentGetResults"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
```

Comment:
> "put": { [](http://example.com/codeflow?start=5&length=9)

There must be PATCH if the api supports a partial update of a resource.

Reply:
We do not support partial update.